eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -0,0 +1,75 @@
+[CmdletBinding()]
+param (
+)
+Set-StrictMode -Version 3
+
+. $PSScriptRoot/ChangedFiles-Functions.ps1
+. $PSScriptRoot/Logging-Functions.ps1
+
+$repoPath = Resolve-Path "$PSScriptRoot/../.."
+$pathsWithErrors = @()
+
+$filesToCheck = @(Get-ChangedSwaggerFiles)
+if (!$filesToCheck) {
+  LogInfo "No swagger files found to check"
```

Comment:
Do you maybe wanna use `OpenAPI` here?

See:
https://github.com/Azure/azure-sdk-tools/issues/6374#issuecomment-1875388701

> Please don’t use the term "swagger" in the text of the PR comments — use "API defintion" or something similar. "Swagger" is a trademark of the SmartBear company and while I know many at Microsoft use it as a synonym for "API definition" I think we should set an example of using correct terminology particularly in public spaces such as the azure-rest-api-specs repo.
>
> [swagger.io/blog/api-strategy/difference-between-swagger-and-openapi](https://swagger.io/blog/api-strategy/difference-between-swagger-and-openapi/)

Reply:
I think I've mentioned this before, but I think using consistent terminology for spec authors is more important than gradually deprecating the term "swagger" and causing confusion by using two different terms for the same thing.  So I'm happy to change from "swagger" to "openapi" or whatever, when we rename all the existing checks.